doc/overview/about_project_capabilities.md

```diff
@@ -86,5 +86,10 @@ It's very important that project capabilities you define fit this criteria:
   - Bad: `VB`
 - May include a version number, when necessary, but is usually discouraged.
 
+### Dynamic project capabilities in Visual Studio .VNext
```

Comment:
Just use Visual Studio Next to keep consistency with the rest of the documentation